### PR TITLE
Vit branch

### PR DESCRIPTION
**Pull request by aratras**
- Added user's ads lists with deleting 
- Added Users API controllers
- Fixed some issues in services

### CODE REVIEW

File: Advertisements.BusinessLogic/Services/AdvertisementService.cs [hunk 1]
```diff
@@ -87,7 +91,21 @@ public void Delete(int id)
...
             }
         }
+        public IEnumerable<AdvertisementDTO> GetByUser(string id)
+        {
+            IEnumerable<Advertisement> Advertisement;
+
+            using (var uow = _uowfactory.CreateUnitOfWork())
+            {
+                var repo = uow.GetRepo<Advertisement>();
+                Advertisement = repo.GetAll().Where(e => e.ApplicationUserId == id);
```
Comment (dmhatescookies) [inline Advertisements.BusinessLogic/Services/AdvertisementService.cs:105]: TODO later write Where method in RepositoryBase

File: Advertisements.BusinessLogic/Services/AspNetUsersService.cs [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            }
+        }
+
+        public AspNetUsersDTO Get(int id)
+        {
+            ApplicationUser ApplicationUser;
+
+            using (var uow = _uowfactory.CreateUnitOfWork())
+            {
+                var repo = uow.GetRepo<ApplicationUser>();
```
Comment (dmhatescookies) [inline Advertisements.BusinessLogic/Services/AspNetUsersService.cs:50]: Include params

**Comment by dmhatescookies** [comment]
Ok, Good

**Comment by Vasiakozak1** [comment]
Ok!

**Pull request merged**